models/issue.go

```diff
@@ -1517,6 +1523,14 @@ func GetUserIssueStats(opts UserIssueStatsOptions) (*IssueStats, error) {
 	if len(opts.IssueIDs) > 0 {
 		cond = cond.And(builder.In("issue.id", opts.IssueIDs))
 	}
+	if opts.IsArchived != util.OptionalBoolNone {
+		relevantRepoIDs := []int64{}
+		r := []*Repository{}
+		s := x.Table("repository").Where(builder.Eq{"is_archived": opts.IsArchived.IsTrue()})
+		s.Select("id").Find(&relevantRepoIDs)
+		s.Find(&r)
```

Comment:
Work it did, as far as I could see, but I get that stylistically my approach is less than ideal. I didn't use `Join` because `builder.Cond` has no such thing.

Let me look into it again and try and do better this time around.